docs/source/data-model.rst

```diff
@@ -1,18 +1,39 @@
-**********
-Data Model
-**********
+********************************
+Data Model: Documents and Events
+********************************
+
+*begin remove*
 
 Bluesky's event-based data model supports complex, asynchronous data collection
 and enables sophisticated live, prompt, streaming, and *post-facto* data
 analysis.
 
+*end remove*
+
+A primary design goal of bluesky is to enable better research by recording
+rich metadata alongside measured data for use in later analysis. Documents are
+how we do this.
+
+A *document* is our term for a Python dictionary with a schema --- that is,
+organized in a
+`formally specified <https://github.com/NSLS-II/event-model>`_ way --- created
+by the RunEngine during plan execution.  All of the metadata and data generated
+by executing the plan is organized into documents.
+
+A :doc:"later section <callbacks>" describes how outside functions can
+"subscribe" to a stream of these documents, visualizing, processing, or saving
+them. This section provides an outline of documents themselves, aiming to give
+a sense of the structure and familiarity with useful components.
+
 Overview
 ========
 
 The data model is composed of six types of Documents, which in Python are
 represented as dictionaries but could be represented as nested mappings (e.g.
 JSON) in any language. Each document class has a defined, but flexible, schema.
 
+*begin remove*
```

Comment:
I'm surprised you marked this for removal. I figure it's good to have a very short summary of the meaning of each document somewhere. As someone coming to this fresh, do you think this is so succinct that it doesn't add much value, that we might as well go straight to the more detailed description?

I would be happy with any of the following:
- Drop bluesky's version of this overview and keep event-model's.
- Merge some of event-model's verbiage (e.g. "the who / why / what") into bluesky's.

It depends on how much detail we want to present this early.

Reply:
I liked the similar summary of each document from documents.rst because there was a lot of detail, but I see the value of a shorter summary for each document. I will drop the bluesky version.